.github/workflows/ci.yaml

```
@@ -11,45 +11,48 @@ on:
       - master
 
 jobs:
```

actionlint (1.7.7)

"jobs" section should not be empty. please remove this section if it's unnecessary

(syntax-check)